src/components/external-apps/ExternalAppService.ts

```diff
@@ -12,6 +12,14 @@ export interface HelmAppDeploymentHistoryResponse extends ResponseType {
     result?: HelmAppDeploymentHistory
 }
 
+export interface HelmAppDeploymentManifestDetail {
+    manifest?: string;
```

Comment:
remove manifest from HelmAppDeploymentDetail as it is not being used now

Reply:
Done